v3/src/components/case-table/use-columns.tsx

```diff
@@ -125,7 +129,7 @@ export const useColumns = ({ data, indexColumn }: IUseColumnsProps) => {
       },
       { name: "useColumns [rebuild columns]", equals: comparer.structural, fireImmediately: true }, data
     )
-  }, [RenderCell, caseMetadata, collectionId, data, indexColumn, parentCollection])
+  }, [RenderCell, caseMetadata, collectionId, data, indexColumn, parentCollection, attrIdToEdit, setAttrIdToEdit])
```

Comment:
Changes to `attrIdToEdit` would result in reinstalling the MobX reaction rather than just having it fire.